content/about/notes/1.0/index.md

```diff
@@ -22,3 +22,8 @@ to usual pile of bug fixes and performance improvements.
   EKS by using [manual injection](/docs/setup/kubernetes/sidecar-injection/#manual-sidecar-injection) for
   sidecars and turning off galley using the [Helm parameter](/docs/setup/kubernetes/helm-install)
   `--set galley.enabled=false`.
+
+- In a [multicluster deployment](/docs/setup/kubernetes/multicluster-install) the mixer-telemetry
+  and mixer-policy components do not connect to the Kubernetes API endpoints of any of the remote
+  clusters.  This results in a loss of telemetry fidelity as some of the metadata associated
```

Comment:
The use of "telemetry" above is not accurate.  The lose of fidelity applies to both telemetry and policy.   I would change this back to the way I had proposed to you privately.